brainstormforce/ultimate-addons-for-gutenberg
# Fix: CF7 & GF-Styler - Category attribute missing for cf7 and gf styler block due to which block visible in uncategorized.

### Description

- NA

### Screenshots

- https://share.getcloudapp.com/X6uKnw4P

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Open editor.
- Verify all block visible in Spectra categories.
- Screencast - https://share.getcloudapp.com/X6uKnw4P

### Checklist:
